recipes/sambamba/meta.yaml

```diff
@@ -11,13 +11,7 @@ source:
   md5: 07d82a2102126565e6796658face7c71 # [osx]
 
 build:
-  number: 0
-
-requirements:
-  build:
-  run:
-    - samtools # required for mpileup
-    - bcftools # required for mpileup
```

Comment:
Does that mean those are no longer needed by sambamba?

Reply:
No. It means they were never really needed by sambamba (depending on how you look at it :-) ). They are only needed for sambamba mpileup, whose help-page also says so. Please see the discussion at  #6974.